text/0000-pack-publish-buildpack.md

```diff
@@ -42,6 +42,102 @@ There are four mechanisms for publishing a buildpack:
 * Direct, which commits directly to the local Git repository for the index and makes a Git push to the origin
 * Yank, which removes a buildpack from the registry
 
+## Configuration
+
+##### Base Schema
```

Comment:
is this for `.pack/config.toml`?

Reply:
Yes.

Reply:
One of the downsides to this proposed change is that it would work more like `git remotes` where you name your registries instead of simply `--buildpacks-registry <URL>` it would be `--buildpacks-registry <name>` and we could have commands like `pack registry add <name> ...`

Reply:
given how few people with change this, i think that's ok.

I think it'll be more common to use the `cnb://<registry-url>/<ns>/<name>` URI in the `--buildpack` or `project.toml` fields.